.travis.yml

```diff
@@ -7,7 +7,7 @@ notifications:
 # Run latest 3.x and 2.x next, to get quick compatibility results.
 # Then run the remainder.
 python:
-  - "pypy"
+  - "pypy-5.3.1"
```

Comment:
If we've got one test that is failing on a specific version of pypy, we should document that at the test and skip it for just that version.  We've done it before, and recently.